ocrd/ocrd/workspace.py

```diff
@@ -281,8 +281,8 @@ def image_from_page(self, page, page_id):
 
         alternative_image = page.get_AlternativeImage()
         if alternative_image:
-            # (e.g. from page-level cropping, binarization, deskewing or despeckling)
-            # assumes implicit cropping (i.e. page_xywh has been applied already)
+            # (e.g. from page-level cutting, binarization, deskewing or despeckling)
```

Comment:
This actually refers to `cropping` the preprocessing step, so
```suggestion
            # (e.g. from page-level cropping, binarization, deskewing or despeckling)
```